infra/resources/_modules/messages_eventhub/messages_eventhub.tf

```
@@ -0,0 +1,38 @@
+module "messages-evh" {
+  source = "github.com/pagopa/dx//infra/modules/azure_event_hub?ref=3f205e62474782678a563f4cff92e479a34feecd"
+
+  environment = {
+    prefix          = var.prefix
+    env_short       = var.env_short
+    location        = var.location
+    domain          = var.domain
+    app_name        = "etl"
+    instance_number = var.instance_number
```

no var needed here :) 

```suggestion
    instance_number = "01"
```